1.0/profile.ttl

```diff
@@ -0,0 +1,63 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    dcterms:identifier "geosparql" ;
+    dcterms:created "2009-10-27"^^xsd:date ;
+    dcterms:modified "2012-07-19"^^xsd:date ;
```

Comment:
The `dcterms:created` and `dcterms:modified` dates seem to apply to the GeoSPARQL specification itself, not its profile. That doesn't seem right.

Reply:
Yes, right again @FransKnibbe. I've changed these properties